code/modules/mob/living/carbon/xenomorph/abilities.dm

```diff
@@ -1205,6 +1205,79 @@
 
 	succeed_activate()
 
+//////////////////////////
+/// Build xeno jel turret
+//////////////////////////
+
+/datum/action/xeno_action/activable/build_turret_jel
+	name = "Secrete jel turret"
```

Comment:
What's Jel?
(Larva jelly presumably, but someone playing ingame might not know that.)